### DISCUSSION TITLE
[해결됨] 키보드가 텍스트뷰 입력을 가리는 것을 어떻게 해결하시나요?

### DISCUSSION THREAD

**Discussion by EuniceNam** [DeveloperAcademy-POSTECH/MacC-GetARock#62] [Q&A] [answered]
모여락 추가 뷰에서, 마지막 입력 항목이 동적으로 높이가 늘어나는 텍스트뷰인데, 텍스트뷰에 입력이 길어지면 키보드가 텍스트뷰를 가리게 됩니다. 

키보드가 커서를 가리는 일을 막는 방법이나 참고 리포지토리를 아신다면 알려주시면 감사합니다.

(당장 적용하기보단 중요 기능구현 후 해볼 생각입니다)

- 머피가 알려준 서드파티: IQKeyboardManager

**Comment by compuTasha** [comment, marked as answer]
참고 링크를 찾았읍니다!

https://seizze.github.io/2019/11/17/iOS에서-키보드에-동적인-스크롤뷰-만들기.html

**Reply by EuniceNam** [reply]
해결되었습니다!

전에 참고했던 페이지인데, 그 때는 ScrollView의 오토레이아웃이 제대로 설정되어있지 않아 잘못 작동했습니다. 
#61 PR의 [알로라의 리뷰](https://github.com/DeveloperAcademy-POSTECH/MacC-GetARock/pull/61#pullrequestreview-1188019330) 반영 8f22d8de9e4b769198ce89c4496c0e0de2b143d6 후 잘 작동합니다. 
감사합니다.

**Reply by compuTasha** [reply]
앗 다행이에요!!! 🥺
